pkg/apis/apps/v1alpha1/description.go

```diff
@@ -62,6 +63,16 @@ type DescriptionSpec struct {
 
 // DescriptionStatus defines the observed state of Description
 type DescriptionStatus struct {
+	// PhaseStatus indicates the Phase and reason of DescriptionPhase
+	// +optional
+	Phase PhaseStatus `json:"phaseStatus,omitempty"`
+
+	// ManifestStatuses contains running status of manifests in spec.
+	// +optional
+	ManifestStatuses []ManifestStatus `json:"manifestStatus,omitempty"`
+}
+
+type PhaseStatus struct {
```

Comment:
No need to nest this struct. And `phase` and `reason` are used to record the deployment status to child clusters. Better not change this, unless there is a need.

Reply:
> No need to nest this struct. And `phase` and `reason` are used to record the deployment status to child clusters. Better not change this, unless there is a need.

fixed this.